tests/unit/backend/chalice/api_server/test_v1_collection.py

```diff
@@ -763,3 +763,42 @@ def test__revision_not_owner__403(self):
         headers = {"host": "localhost", "Content-Type": "application/json", "Cookie": get_auth_token(self.app)}
         response = self.app.post(test_url, headers=headers)
         self.assertEqual(403, response.status_code)
+
+    def test__publish_revision__201(self):
```

Comment:
Could you add links to the original collection and check that one of them is copied over and one of them is deleted to ensure `remove_relationships=True` is working as expected?